app/assets/javascripts/application.js

```diff
@@ -136,6 +134,15 @@ function onContentLoad(){
   })
 }
 
+function activateDatatables() {
+  $('[data-table=inline]').not('.dataTable').dataTable(
+      {
+        "sDom": "<'row'<'col-md-6'f>r>t<'row'<'col-md-6'i><'col-md-6'p>>",
+        "sPaginationType": "bootstrap"
+      }
+  );
+}
```

Comment:
Looks like in my tab, I will need completely different datatable initialization, can you rename the key from `inline` to something more specific, so my table can use it's own key?